aspnet/whitepapers/aspnet-and-iis6.md

```diff
@@ -71,27 +71,27 @@ If you do not wish to use the 'Configure Your Server Wizard' you can optionally
 
 First open the Control Panel:
 
-![](aspnet-and-iis6/_static/image8.jpg)
+![Screenshot of the add or remove programs screen with the M S D N Library for Visual Studio dot net 2003 option highlighted.](aspnet-and-iis6/_static/image8.jpg)
 
 Next, click on 'Add/Remove Windows Components' which will open the 'Windows Components Wizard':
 
-![](aspnet-and-iis6/_static/image9.jpg)
+![Screenshot of  the Windows components wizard windows components screen. A S P dot net is highlighted.](aspnet-and-iis6/_static/image9.jpg)
```

Comment:
Suggestion: Screenshot of the Windows components wizard window. A S P dot net is highlighted.